test/libsolidity/syntaxTests/metaTypes/type_max_from_ternary_expression.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    function max(bool isUint) public returns (uint8) {
+        return (isUint ? type(uint8) : type(int8)).max;
```

Comment:
Is `(type(uint8)).max` also affected by the change?

Reply:
No, that still works.

Reply:
Let's add a test case then. These things should be visible in tests.